### PR TITLE
Update staff.js

### PR DESCRIPTION
**Pull request by Derrick355**
Added myself. I did it for you, jackass.

### CODE REVIEW

**Comment by MY5TERIO** [comment]
Fix it dipshit

File: js/staff.js [hunk 1]
```diff
@@ -107,6 +108,10 @@ placeholders.setValue('page', "Staff");
         "https://cdn.discordapp.com/avatars/297636982723444736/23dc2b50585536196edaef043dbb7fc0.png?size=2048",
         "I don't know what to put here",
         [StaffCategory.DONATORS])
+        .add("iPlay G",
+            "https://cdn.discordapp.com/avatars/146151306216734720/eeeff7ae442729e69850a23c8f0330ab.png?size=128 ",
+            "Resident Jackass",
+             [Staff.Category.JACKASS])
```
Comment (Jacxk) [inline js/staff.js]: That dont work because it should be `StaffCategory` and not `Staff.Category`

Comment (MY5TERIO) [reply js/staff.js]: Pleaze fix I'm on mobile

Comment (Jacxk) [reply js/staff.js]: dont worry about it, for some reason its shows up fine in master

**Pull request merged**